Remove phpdocsinvalidtag, phpdocsnotrecommendedtag, phpdocsinvalidpathtag

andrewnicols opened the pull request:
Replaced with https://github.com/moodlehq/moodle-cs/pull/124

stronk7 reviewed (commented):
Maybe we should consider also removing the config arrays, if they aren't used anymore?

Link: https://github.com/moodlehq/moodle-local_moodlecheck/blob/main/file.php#L1097-L1217

I'm happy either way, your preference.

Ciao :-)

andrewnicols commented:
> Maybe we should consider also removing the config arrays, if they aren't used anymore?
> 
> Link: https://github.com/moodlehq/moodle-local_moodlecheck/blob/main/file.php#L1097-L1217
> 
> I'm happy either way, your preference.
> 
> Ciao :-)

Ah good catch

The validTags are still in use for inline params -- I haven't moved that one yet, but I've removed the rest.
I've also removed some unused fixtures and raised a new issue for other fixtures which should have been removed.

stronk7 reviewed (approved):
All ok, landing this before https://github.com/moodlehq/moodle-cs/pull/124 that is near, near, near... thanks!
